run-health-checks.sh

```diff
@@ -45,10 +45,10 @@ fi
 
 #add accelerated network if applicable
 NHC_DIR=$(dirname "${BASH_SOURCE[0]}")
-acc_net=$(ibstatus mlx5_an0)
 acc_file="$NHC_DIR/conf/$conf_name.conf"
+acc_net=$(ibstatus mlx5_an0)
```

Comment:
This change is un-needed

Reply:
The change here is the order that the commands get executed.

Reply:
I added "ibstatus_exit_code=$?" to make it clear we want the return valueof that command. the 'acc_net' variable is assigned so output of ibstat is redirected to the variable instead of stdout